node/pkg/node/node_test.go

```diff
@@ -250,6 +315,66 @@ func someMessage() *common.MessagePublication {
 	}
 }
 
+var tokenBridgeSequenceCounter uint64 = 0
+
+func governedMsg(shouldBeDelayed bool) *common.MessagePublication {
+	buildMockTransferPayloadBytes := func(
+		tokenChainID vaa.ChainID,
+		tokenAddrStr string,
+		toChainID vaa.ChainID,
+		toAddrStr string,
+		amtFloat float64,
+	) []byte {
+		bytes := make([]byte, 101)
+		bytes[0] = 1 // tb payload type
+
+		amtBigFloat := big.NewFloat(amtFloat)
+		amtBigFloat = amtBigFloat.Mul(amtBigFloat, big.NewFloat(100000000))
+		amount, _ := amtBigFloat.Int(nil)
+		amtBytes := amount.Bytes()
+		if len(amtBytes) > 32 {
+			panic("amount will not fit in 32 bytes!")
+		}
+		copy(bytes[33-len(amtBytes):33], amtBytes)
+
+		tokenAddr, _ := vaa.StringToAddress(tokenAddrStr)
+		copy(bytes[33:65], tokenAddr.Bytes())
+		binary.BigEndian.PutUint16(bytes[65:67], uint16(tokenChainID))
+		toAddr, _ := vaa.StringToAddress(toAddrStr)
+		copy(bytes[67:99], toAddr.Bytes())
+		binary.BigEndian.PutUint16(bytes[99:101], uint16(toChainID))
+		return bytes
```

Comment:
Some comments for this would be really nice. It is not intuitive what it is doing if you lack the context of working on the code a bunch.